ckan_cloud_operator/providers/ckan/env/manager.py

```diff
@@ -163,6 +163,13 @@ def _write_yaml(data):
     _file.close()
 
 
+def _read_yaml():
+    _file = open(_get_config_file_name(), "r")
+    yaml_conf = yaml.load(_file, Loader=yaml.FullLoader)
+    _file.close()
+    return yaml_conf
```

Comment:
```suggestion
def _read_yaml():
    with open(_get_config_file_name()) as _file:
        yaml_conf = yaml.load(_file, Loader=yaml.FullLoader)
    return yaml_conf
```

It is always suggested as good practice to use `with` statement when reading files, also "r" is the default mode so not need to add it :)